### PR TITLE
New guide: HPE ProLiant Agentless Management Service

### PR DESCRIPTION
**Pull request by neelchauhan**
#### Author checklist (Completed by original Author)
- [X] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [X] If applicable, steps and instructions have been tested to work
- [X] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

**Comment by neelchauhan** [comment]
As a note, I'm not sure if `automation` is the correct place to put this, but I didn't see another better section to put this, so hence I used `automation`.

**Review by sspencerwire — APPROVED** [review]
@neelchauhan Thank you! This seems very brand-specific, BUT useful none-the-less.

**Comment by sspencerwire** [comment]
> As a note, I'm not sure if `automation` is the correct place to put this, but I didn't see another better section to put this, so hence I used `automation`.

Message me on Mattermost: Perhaps we could create a new section. What would you suggest? I'm traveling, so even though I've approved this PR for merging, I'll wait a bit for your ideas.

File: docs/guides/automation/hpe_amsd.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+systemctl enable --now amsd
+```
+
+If you want to check if `amsd` is working, log into iLO via your web browser. If installed correctly, iLO should report our server is running Rocky Linux:
+
+![HPE iLO showing Rocky Linux 9.3](../images/hpe_ilo_amsd.png)
+
+## Conclusion
+
+A common criticism of HPE servers are high fan speeds when using third-party components such as SSDs or other add-in PCI Express cards which aren't officially sanctioned by HPE (e.g. video capture cards). Even if you only use HPE-branded components, using `amsd` allows HPE ProLiant servers to run cooler and quieter than just using Rocky Linux alone on one.
```
Comment (SergeCroise) [inline docs/guides/automation/hpe_amsd.md:59]:
```suggestion
A common criticism of HPE servers are high fan speeds when using third-party components such as SSDs or other add-in PCI Express cards which aren't officially sanctioned by HPE (e.g., video capture cards). Even if you only use HPE-branded components, using `amsd` allows HPE ProLiant servers to run cooler and quieter than just using Rocky Linux alone on one.
```

**Review by SergeCroise — COMMENTED** [review]
Punctuation according to this example:
https://www.merriam-webster.com/dictionary/e.g.#examples

File: docs/guides/automation/hpe_amsd.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  - automation
+---
+
+# HPE ProLiant Agentless Management Service
+
+## Introduction
+
+HPE ProLiant servers have a companion software called Agentless Management Service which [according to HPE](https://techlibrary.hpe.com/docs/iss/EL8000t/setup_install/GUID-1CF69B20-790A-4EDC-A162-9D64572ED9E8.html) "uses out-of-band communication for increased security and stability." In addition, "with Agentless Management, health monitoring and alerting is built into the system and begins working the moment auxiliary power is connected to the [your server]".
+
+As an example, this is used to reduce fan speeds on a HPE ProLiant ML110 Gen11 in the author's home lab.
```
Comment (SergeCroise) [inline docs/guides/automation/hpe_amsd.md:16]: a HPE Proliant...
or 
an HPE Proliant...
?

**Comment by neelchauhan** [comment]
> Message me on Mattermost: Perhaps we could create a new section. What would you suggest? I'm traveling, so even though I've approved this PR for merging, I'll wait a bit for your ideas.

I'd like a `hardware` section for hardware-specific guides. Not just HPE, but also Dell, Raspberry Pi, Apple, et al.

I took a while to respond because I'm moving back to NYC and was busy packing.

File: docs/guides/automation/hpe_amsd.md (hunk 2 above)
Comment (neelchauhan) [reply docs/guides/automation/hpe_amsd.md]: Done!

File: docs/guides/automation/hpe_amsd.md (hunk 1 above)
Comment (neelchauhan) [reply docs/guides/automation/hpe_amsd.md]: Done!

**Review by sspencerwire — APPROVED** [review]

**Pull request merged**